app/templates/components/nav-bar.hbs

```diff
@@ -47,32 +47,32 @@
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
         <div class="item">
-          <a class="ui blue button" style="text-transform: uppercase; font-weight: 400" href="{{href-to 'create'}}">
+          {{#link-to 'create' class='ui blue button'}}
             {{t 'Create Event'}}
-          </a>
+          {{/link-to}}
         </div>
       {{/if}}
     </div>
   </div>
 </div>
 <div class="mobile only row">
-  <div class="ui navbar {{if device.isMobile 'tiny'}} menu">
+  <div class="ui navbar tiny menu">
     <a class="menu item open sidebar">
       <i class="large content icon"></i>
     </a>
     {{#link-to 'index' class='item' activeClass=''}}
-      <div class="ui header {{if device.isMobile 'small text'}}">{{config.appName}}</div>
+      <div class="ui header small text">{{config.appName}}</div>
     {{/link-to}}
-    <div class="right menu">
+    <div class="right menu nav-bar">
       {{#if session.isAuthenticated}}
         <a class="notification item">
           {{notification-dropdown notifications=notifications}}
         </a>
       {{/if}}
       <div class="item">
-        <a class="ui blue button button-text {{if device.isMobile 'small text'}}" style="text-transform: uppercase; font-weight: 400;" href="{{href-to 'create'}}">
+        {{#link-to 'create' class='ui blue button button-text small text'}}
```

Comment:
@CosmicCoder96 yes you are right I removed that earlier and it was a very silly mistake by me I will be more careful in future.thanks!!